src/ObjCRuntime/PlatformAvailability2.cs

```diff
@@ -38,7 +38,6 @@ public enum PlatformName : byte
 		WatchOS,
 		TvOS,
 		MacCatalyst,
-		UIKitForMac = MacCatalyst, // temporary
```

Comment:
hmm... it's public so it can break source compatibility (not binary compatibility since it's an enum)
maybe it should be `[Obsolete]` ?